styles/styles.css

```diff
@@ -0,0 +1,67 @@
+@import url('styles/fonts.css');
+
+.advantages {
+    width: 1440px;
+    margin: 0 auto;
+    padding: 120px 16px 120px;
+}
+
+.title {
+    text-align: center;
+    font-family: 'Roboto medium', sans-serif;
+    font-size: 40px;
+}
+
+.line {
+    max-width: 1440px;
+    width: 100px;
+    height: 4px;
+    background: #f5bd41;
+    margin: 20px auto 85px;
+}
+
+.row {
+    display: flex;
+    gap: 16px;
+    justify-content: center;
+}
+
+.col {
+    white-space: nowrap;
+    width: 25%;
+
+    text-align: center;
+    font-family: 'Roboto regular', sans-serif;
+    font-size: 24px;
+}
+
+.icons {
+    height: 100px;
+    background-size: cover;
+    margin-bottom: 30px;
+}
+
+
+.footer {
+    margin: 0 auto;
+    background: #10367C;
+}
+
+.menu-row {
+    max-width: 1440px;
```

Comment:
as you can notice on design all content is align to the center. So you need to align your footer content margin:0 auto